cypress.config.js

```diff
@@ -10,10 +10,26 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
```

Comment:
The random number generation logic is incorrect. The `Math.random(1000)` should be `Math.random()` to generate a number between 0 and 1, which is then multiplied by 1000.